lib/package_convenience.rb

```diff
@@ -0,0 +1,33 @@
+require 'json'
+require_relative 'const'
+
+class PackageFunctions
```

Comment:
Maybe a shorter name is better for classes? Like `PackageUtils`

Reply:
Good idea and the class name should also match the filename to avoid confusion.